packages/mui-material/src/Slider/Slider.d.ts

```diff
@@ -70,19 +86,54 @@ export type SliderTypeMap<
       thumbColorSecondary?: string;
       /** Class name applied to the thumb element if `size="small"`. */
       thumbSizeSmall?: string;
+      /** Class name applied to the thumb label element. */
+      valueLabel: string;
+      /** Class name applied to the thumb label element if it's open. */
+      valueLabelOpen: string;
+      /** Class name applied to the thumb label's circle element. */
+      valueLabelCircle: string;
+      /** Class name applied to the thumb label's label element. */
+      valueLabelLabel: string;
     };
     /**
      * The size of the slider.
      * @default 'medium'
      */
     size?: OverridableStringUnion<'small' | 'medium', SliderPropsSizeOverrides>;
+    /**
+     * The props used for each slot inside the Slider.
+     * @default {}
+     */
+    slotProps?: {
+      root?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      track?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      rail?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      thumb?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      mark?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      markLabel?: SlotComponentProps<'span', SliderComponentsPropsOverrides, SliderOwnerState>;
+      valueLabel?: SlotComponentProps<
+        typeof SliderValueLabelComponent,
+        SliderComponentsPropsOverrides,
+        SliderOwnerState
+      >;
+      input?: SlotComponentProps<'input', SliderComponentsPropsOverrides, SliderOwnerState>;
+    };
     /**
      * The system prop that allows defining system overrides as well as additional CSS styles.
      */
     sx?: SxProps<Theme>;
-  };
+    /**
+     * Controls when the value label is displayed:
+     *
+     * - `auto` the value label will display when the thumb is hovered or focused.
+     * - `on` will display persistently.
+     * - `off` will never display.
+     * @default 'off'
+     */
+    valueLabelDisplay?: 'on' | 'auto' | 'off';
+  } & Omit<SliderUnstyledTypeMap['props'], 'slotProps' | 'isRtl'>;
```

Comment:
I wonder whether we should not extend `SliderUnstyled` types now that the component is not inherited. We can replicate the types here. 

If there is some change in types in `SliderUnstyled` later on, we may miss that it will affect Material UI Slider, assuming that the component is no longer inherited in Material UI.

Reply:
Yes, indeed. There shouldn't be any references to SliderUnstyled

Reply:
@michaldudak SliderUnstyled references were removed by allowing Slider to have its own types. 